SCF-Session-8-Assignment/src/question/Mammal.java

```diff
@@ -0,0 +1,18 @@
+package question;
+
+/**
+ * This class is for category Mammal
+ * @author Jugal
+ * Dated 07/23/2019
+ */
+public abstract class Mammal extends Animal{
+

```

Comment:
Declare some different properties of Mammal class and initialize them in Mammal class constructor and initialize Animal class property using super().

Reply:
Done